installation.md

```diff
@@ -8,7 +8,7 @@ To use clangd, you need:
 
 ## Installing clangd
 
-You'll want a **recent** version of clangd. The current release is 11.0.
+You'll want a **recent** version of clangd. The current release is 12.0.
 
 After installing, `clangd --version` should print `clangd version 7.0.0` or later.
```

Comment:
This should also be "`clangd version 12.0.1` or later".

Reply:
Sure, updated. Thanks!